src/modules/ModuleLinker.v3

```diff
@@ -16,6 +16,9 @@ class ModuleLinker(engine: Engine,
 	private def deferred_imports = Array<Array<int>>.new(named_modules.length);
 
     def link() -> Array<Instantiator> {
+		// Expose the wizeng HostModule if it's an activated extension
+		if (engine.extensions.WIZENG) ModuleRegistry.expose("wizeng");
```

Comment:
I'm not sure what you meant by adding the wizeng HostModule to the linking context [here](https://github.com/titzer/wizard-engine/pull/245#discussion_r1890905079), I pushed it down into the linker.link() logic in case this is what you meant.